test/integration/analysis-controller/install.yaml

```diff
@@ -1,14 +1,48 @@
 apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisValueTemplate
+metadata:
+  name: ready
+  namespace: $NAMESPACE
+spec:
+  provider:
+    name: my-mocked-provider
+    namespace: $NAMESPACE
+  query: 'sum(kube_pod_container_status_ready{namespace="{{.ns}}"})'
+---
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: AnalysisDefinition
+metadata:
+  name: ed-my-proj-dev-svc1
+  namespace: $NAMESPACE
+spec:
+  objectives:
+    - analysisValueTemplateRef:
+        name: ready
+        namespace: $NAMESPACE
+      target:
+        failure:
+          lessThan:
+            fixedValue: 2
+        warning:
+          lessThan:
+            fixedValue: 3
+      weight: 1
+      keyObjective: false
+  totalScore:
+    passPercentage: 90
+    warningPercentage: 75
+---
+apiVersion: metrics.keptn.sh/v1alpha3
 kind: Analysis
 metadata:
   name: analysis-sample
   namespace: $NAMESPACE
 spec:
   timeframe:
-    from: $CURRENT_TIME
-    to: $LATER
+    from: 2023-09-14T07:33:19Z
```

Comment:
this can be any time frame now (just needs to be a valid timestamp), as we are mocking the prometheus response and don't need actual monitoring data to be available